documentation/user-consent.md

```diff
@@ -48,122 +43,269 @@ The document details aspects regarding the consent management, which includes fo
+    - Off-net scenarios using refresh_token, as long as there was a connection when the first access_token was requested.
+
+
+>[TO BE EDITED/COMPLETED] - [Technical disscussion on ruleset to be applied](https://github.com/camaraproject/IdentityAndConsentManagement/issues/52) are ongoing.
+
+#### CIBA flow (Backend flow)
+
+```mermaid
+sequenceDiagram
+autonumber
+title Consume a Hyperscaler Enhanced Service API - CIBA flow
+participant User as End User<br>@Authentication Device
+participant FE as Device App<br> (Consumption Device)
+participant BE as Invoker (App Backend/Aggregator)  
+box Operator
+  participant ExpO as OGW Platform<br>@Operator  
+  participant Consent as Consent Master<br>@Operator
+end
+
+Note over FE,BE: Feature needing<br>Operator Capability  
+Note over BE: Select User Identifier:<br> Ip:port / MSISDN / other TBD  
+
+alt OIDC Client-Initiated Backchannel Authentication (CIBA) Standard Flow between Invoker and Operator.
+  BE->>+ExpO: POST /bc-authorize<br> Credentials, purpose,<br>login_hint including User Identifier    
+  ExpO->>ExpO: - Validate User Identifier<br>- (Opt) map to Telco Identifier e.g.: phone_number<br>- Set UserId (sub)  
+  ExpO->>ExpO: Check legal basis of the purpose<br> e.g.: contract, legitimate_interest, consent, etc  
+  opt If User Consent is required for the legal basis of the purpose  
+    ExpO->>Consent: Check if Consent is granted
+  end
+  alt If Consent is Granted or Consent not needed for legal basis   
+    ExpO->>BE: HTTP 200 OK {"auth_req_id": "{OperatorAuthReqId}"  
+  else If Consent is needed and is NOT granted - Out Of Band Consent Capture (Push/SMS/other)
+    note over ExpO,User: User Interaction <br> out-of-band capture consent mechanism chosen by the Operator
+    ExpO->>BE: HTTP 200 OK {"auth_req_id": "{OperatorAuthReqId}"
+  end
+  loop Invoker polls until consent is granted or until expires. If granted in advance, token returned in first poll
+    BE->>+ExpO: POST /token <br>Credentials}<br>auth_req_id={OperatorAuthReqId}    
+    ExpO->>-BE: HTTP 200 OK <br>{"access_token": "{OperatorAccessToken}"} 
+  end  
+end
+BE->>ExpO: Access Operator CAMARA API <br> Authorization: Bearer {OperatorAccessToken}    
+ExpO->>ExpO: Decrypt OperatorAccessToken,<br>grants Access,<br>progresses request to API Backend,<br>gets API response  
+ExpO->>BE: CAMARA API Response
+Note over BE,FE: Response
+```
 
->[TO BE EDITED/COMPLETED] - Flow to obtain an access token to consume network API...
+<br>
 
-#### Client Credentials flow
+**Flow description**:
 
->[TO BE EDITED/COMPLETED] - Flow to obtain an access token to consume network API...
+First, the API invoker (which could potentially be the application BE, an aggregator, etc.) requests a 3-legged access token to the operator OGW platform. The process follows the OpenID Connect [Client-Initiated Backchannel Authentication (CIBA)](https://openid.net/specs/openid-client-initiated-backchannel-authentication-core-1_0.html) flow.
```

Comment:
```suggestion
First, the API invoker (which could potentially be the Application Backend, an Aggregator, etc.) requests a 3-legged access token to the operator API Gateway Platform. The process follows the OpenID Connect [Client-Initiated Backchannel Authentication (CIBA)](https://openid.net/specs/openid-client-initiated-backchannel-authentication-core-1_0.html) flow.
```

Reply:
Fixed in https://github.com/camaraproject/IdentityAndConsentManagement/pull/54/commits/c7f9715fa8c597e4c9f94e8f0bbf3aecc501ead5.